doc/_layouts/website/page.html

```diff
@@ -2,6 +2,68 @@
 
 {% block head %}
     <link rel="shortcut icon" href="../imgages/favicon.ico">
+	<!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-71727855-2"></script>
+    <link rel="stylesheet" type="text/css" href="//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.1.0/cookieconsent.min.css" />
+	<script src="//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.1.0/cookieconsent.min.js"></script>
+	<script>
+        var setAllowCookies = function(allow) {
+            if (allow) {
+                initGoogleAnalytics();
+            } else {
+                deleteGoogleCookies();
+            }
+        };
+
+        var initGoogleAnalytics = function() {
+            window.dataLayer = window.dataLayer || [];
+            function gtag() { 
+                dataLayer.push(arguments); 
+            }
+            gtag('js', new Date());
+            gtag('config', 'UA-71727855-2');
+        }
+
+        var deleteGoogleCookies = function() {
+            var cookie = document.cookie
+            if (cookie) {
+                cookie.split(";").forEach(function(c) { 
+                    var trimmed = c.replace(/^ +/, "")
+                    if(trimmed.startsWith('_g')) {
+                        document.cookie = trimmed.replace(/=.*/, "=;expires=" + new Date().toUTCString() + ";path=/"); 
+                    }
+                });
+            }
+        }
+
+        window.addEventListener("load", function() {
+            window.cookieconsent.initialise({
+                palette: {
+                    popup: {
+                        background: "#333"
+                    },
+                    button: {
+                        background: "#f1d600"
+                    }
+                },
+                type: "opt-in",
+                content: {
+                    href: "https://www.gitpod.io/privacy.html#details-III"
+                },
+                onInitialise: function(status) {
+                    setAllowCookies(status === 'allow');
+                },
+                onStatusChange: function(status) {
+                    setAllowCookies(status === 'allow');
+                },
+                cookie: {
```

Comment:
These settings make it impossible to test locally (localhost, http), as we want the same cookie as in the docs (domain) and only on http (secure).